bundles/org.openhab.binding.modbus.lambda/src/main/java/org/openhab/binding/modbus/lambda/internal/handler/GeneralHandler.java

```diff
@@ -0,0 +1,546 @@
+        if (RefreshType.REFRESH == command) {
+            String groupId = channelUID.getGroupId();
+            if (groupId != null) {
+                AbstractBasePoller poller;
+                switch (groupId) {
+                    case GROUP_GENERAL_AMBIENT:
+                        poller = ambientPoller;
+                        break;
+                    case GROUP_GENERAL_EMANAGER:
+                        poller = emanagerPoller;
+                        break;
+                    default:
+                        poller = null;
+                        break;
+                }
+                if (poller != null) {
+                    poller.poll();
+                }
+            }
+        } else {
+            try {
+                if (GROUP_GENERAL_AMBIENT.equals(channelUID.getGroupId())) {
+                    switch (channelUID.getIdWithoutGroup()) {
+                        case CHANNEL_ACTUAL_AMBIENT_TEMPERATURE:
+                            writeInt16(2, getInt16Value(command));
+                            break;
+                    }
+                }
+
+                if (GROUP_GENERAL_EMANAGER.equals(channelUID.getGroupId())) {
+                    switch (channelUID.getIdWithoutGroup()) {
+                        case CHANNEL_EMANAGER_ACTUAL_POWER:
+                            writeInt16(102, getInt16Value(command));
+                            break;
+                        case CHANNEL_EMANAGER_ACTUAL_POWER_SIGNED:
+                            writeInt16(102, getInt16Value(command));
+                            break;
+                    }
+                }
+            }
+
+            catch (LambdaException error) {
+                if (hasConfigurationError() || getThing().getStatus() == ThingStatus.OFFLINE) {
+                    return;
+                }
+                String cls = error.getClass().getName();
+                String msg = error.getMessage();
+
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                        String.format("Error with: %s: %s", cls, msg));
+            }
+        }
+    }
+
+    /**
+     * Initialization: Load the config object of the block Connect to the slave
+     * bridge Start the periodic polling
+     */
+    @Override
+    public void initialize() {
+        config = getConfigAs(GeneralConfiguration.class);
+
+        logger.debug("Initializing thing with properties: {}", thing.getProperties());
+
+        startUp();
+    }
+
+    /**
+     * This method starts the operation of this handler Connect to the slave bridge
+     * Start the periodic polling1
+     */
+    private void startUp() {
+        if (comms != null) {
+            return;
+        }
+
+        ModbusEndpointThingHandler slaveEndpointThingHandler = getEndpointThingHandler();
+        if (slaveEndpointThingHandler == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE, "Bridge is offline");
```

Comment:
The detail does not add any value and can be omitted
```suggestion
            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
```